rat-trap.patch

```diff
@@ -0,0 +1,321 @@
+diff --git a/ConfigAndStart.sqf b/ConfigAndStart.sqf
+index 5d0eebe..360b43d 100644
+--- a/ConfigAndStart.sqf
++++ b/ConfigAndStart.sqf
+@@ -12,6 +12,17 @@
+ _parameters = [
+ 	["SIDE", civilian],
+ 	["VEHICLES", [
++	"C_Offroad_01_F",
++	"C_Offroad_01_repair_F",
++	"C_Hatchback_01_F",
++	"C_Hatchback_01_sport_F",
++	"C_SUV_01_F",
++	"C_Van_01_transport_F",
++	"C_Van_01_box_F",
++	"C_Van_01_fuel_F",
++
++	"C_Truck_02_fuel_F",
++	"C_Truck_02_box_F",
+ 
+ 
+ 	"RDS_Gaz24_Civ_01",
+@@ -28,33 +39,36 @@ _parameters = [
+ 	"RDS_Ikarus_Civ_02",
+ 	"RDS_Ikarus_Civ_02",
+ 
++	"RDS_S1203_Civ_01",
+ 	"RDS_S1203_Civ_02",
++	"RDS_S1203_Civ_03",
+ 
+ 	"RDS_Octavia_Civ_01",
+ 
++	"RDS_Lada_Civ_01",
+ 	"RDS_Lada_Civ_02",
+ 	"RDS_Lada_Civ_03",
+-	"RDS_Lada_Civ_04",
++
+ 	"RDS_Lada_Civ_05",
++	"RDS_Lada_Civ_01",
+ 	"RDS_Lada_Civ_02",
+ 	"RDS_Lada_Civ_03",
+-	"RDS_Lada_Civ_04",
+ 
+-	"RDS_Zetor6945_Base",
++
+ 
+ 	"RHS_Ural_Civ_01",
+ 	"RHS_Ural_Civ_02",
+ 	"RHS_Ural_Civ_03",
+ 	"RHS_Ural_Open_Civ_01",
+ 	"RHS_Ural_Open_Civ_02",
+-	"RHS_Ural_Open_Civ_03",
+-	"rhsgref_nat_uaz"
++	"RHS_Ural_Open_Civ_03"
+ 	]],
+ 	["VEHICLES_COUNT", 10],
+-	["MIN_SPAWN_DISTANCE", 1500],
+-	["MAX_SPAWN_DISTANCE", 2500],
+-	["MIN_SKILL", 1],
+-	["MAX_SKILL", 1],
++	["MIN_SPAWN_DISTANCE", 800],
```

Comment:
800 is quite close for minimum spawn distance. probably enough if no air is involved